airbyte-ci/connectors/pipelines/pipelines/dagger/actions/python/common.py

```diff
@@ -248,9 +248,9 @@ async def apply_python_development_overrides(context: ConnectorContext, connecto
         context.logger.info(f"Mounting CDK from {directory_to_mount}")
 
         # Install the airbyte-cdk package from the local directory
-        # We use --no-deps to avoid conflicts with the airbyte-cdk version required by the connector
+        # We use `--force-reinstall` to use local CDK with the latest updates and dependencies
         connector_container = connector_container.with_mounted_directory(f"/{path_to_cdk}", directory_to_mount).with_exec(
-            ["pip", "install", "--no-deps", f"/{path_to_cdk}"], skip_entrypoint=True
+            ["pip", "install", "--force-reinstall", f"/{path_to_cdk}"], skip_entrypoint=True
```

Comment:
Should it _still_ have both `--force-reinstall` and `--no-deps`?

Reply:
I don't think so, as they're contradictory
```

--no-deps
Don’t install package dependencies.

--force-reinstall
Reinstall all packages even if they are already up-to-date.
```